content/en/blog/2024/steering-results/index.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Istio's Steering Committee for 2024"
+description: Announcing the newest members and a proposed change to election timing.
+publishdate: 2024-02-15
+attribution: "Craig Box, for the Istio Steering Committee"
+keywords: [istio,steering,governance,community,election]
+---
+
+The Istio Steering Committee oversees the administrative aspects of the project, including governance, branding, marketing, and working with the CNCF.
+
+Every year, the leaders in the Istio project [estimate the proportion](https://github.com/istio/community/blob/master/steering/CONTRIBUTION-FORMULA.md) of the [hundreds of companies that have contributed to Istio in the past year](https://istio.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=contributions), and uses that metric to proportionally allocate nine Contribution Seats on our Steering Committee.
+
+Then, four Community Seats are voted for by our project members, with candidates being from companies that did not receive Contribution Seats.
```

Comment:
Do we want to add that these 4 seats are 4 different companies?

Reply:
Maybe - that's always true by design, as you can only hold one Community Seat per company.